### PR TITLE
 #75971 KOMBIT VA-119: Customize metadata

### PR DESCRIPTION
**Pull request by dqtglobeteam**
#78560 [Github] Upload SamlMetadataCustomConfiguration sample to guide user to update his "SAML metadata custom configuration"

### CODE REVIEW

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json [hunk 1]
```diff
@@ -0,0 +1,380 @@
+{
+    "nemlogin2": {
```
Comment (thuannguy) [inline CustomMetadata/SamlMetadataCustomConfiguration Example.json:2]: nemlogin2 does not support aes gcm or those required attributes. You can remove nemlogin2 from this example.

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+            "http://www.w3.org/2009/xmlenc11#aes256-gcm"
+        ]
+    },
+    "kombit": {
+        "IncludeArtifactEndpoints": true,
+        "IncludeAuthenticationContextMethodClasses": true,
+        "IncludeContactPerson": true,
+        "NameIdentifierFormats": [
+            "urn:oasis:names:tc:SAML:2.0:nameid-format:transient",
+            "urn:oasis:names:tc:SAML:2.0:nameid-format:persistent"
```
Comment (thuannguy) [inline CustomMetadata/SamlMetadataCustomConfiguration Example.json:297]: kombit uses X509SubjectName

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+        },
+        "EncryptionAlgorithms": [
+            "http://www.w3.org/2001/04/xmlenc#aes128-cbc",
+            "http://www.w3.org/2001/04/xmlenc#aes256-cbc",
+            "http://www.w3.org/2009/xmlenc11#aes128-gcm",
+            "http://www.w3.org/2009/xmlenc11#aes192-gcm",
+            "http://www.w3.org/2009/xmlenc11#aes256-gcm"
+        ],
+        "UseCustomContactPerson": true,
+        "CustomContactPerson": {
```
Comment (thuannguy) [inline CustomMetadata/SamlMetadataCustomConfiguration Example.json:359]: Where is the technical role?

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+    },
+    "kombit": {
+        "IncludeArtifactEndpoints": true,
+        "IncludeAuthenticationContextMethodClasses": true,
+        "IncludeContactPerson": true,
+        "NameIdentifierFormats": [
+            "urn:oasis:names:tc:SAML:2.0:nameid-format:transient",
+            "urn:oasis:names:tc:SAML:2.0:nameid-format:persistent"
+        ],
+        "IdentityProviderSupportedAttributes": [
```
Comment (thuannguy) [inline CustomMetadata/SamlMetadataCustomConfiguration Example.json:299]: This example is wrong. There are a list of 5 or 6 required claims and each of them must be an element of the IdentityProviderSupportedAttributes array (similar to that of the nemlogin3 node). You can ask NGT for the list of claim.

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+                "AttributeValueXsiType": "http://www.w3.org/2001/XMLSchema#string",
+                "Values": [
+                    "https://data.gov.dk/model/core/specVersion",
+                    "https://data.gov.dk/concept/core/nsis/loa",
+                    "https://data.gov.dk/model/core/eid/professional/rid",
+                    "https://data.gov.dk/model/core/eid/professional/orgName"
+                ]
+            }
+        ],
+        "ServiceProviderRequestedAttributes": [
```
Comment (thuannguy) [inline CustomMetadata/SamlMetadataCustomConfiguration Example.json:314]: Same issues as the IdentityProviderSupportedAttributes  list

**Review by thuannguy — CHANGES_REQUESTED** [review]

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json (hunk 3 above)
Comment (dqtglobeteam) [reply CustomMetadata/SamlMetadataCustomConfiguration Example.json:359]: Fixed

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json (hunk 1 above)
Comment (dqtglobeteam) [reply CustomMetadata/SamlMetadataCustomConfiguration Example.json:2]: remove

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json (hunk 2 above)
Comment (dqtglobeteam) [reply CustomMetadata/SamlMetadataCustomConfiguration Example.json:297]: Fixed

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json (hunk 4 above)
Comment (dqtglobeteam) [reply CustomMetadata/SamlMetadataCustomConfiguration Example.json:299]: Fixed

File: CustomMetadata/SamlMetadataCustomConfiguration Example.json (hunk 5 above)
Comment (dqtglobeteam) [reply CustomMetadata/SamlMetadataCustomConfiguration Example.json:314]: Fixed

**Review by thuannguy — APPROVED** [review]

**Pull request merged**